### DISCUSSION TITLE
Show Geometry Slices in 2D Views

### DISCUSSION THREAD

**Discussion by thogerig** [Kitware/VolView#392] [Ideas]
Dear VolView Team,

I'm genuinely impressed with the VolView project and would like to thank for your great engineering effort.

As I explore the project further, I'm aiming to visualize slices of the geometry data. I believe using a _vtkCutter_ or 
_SlicedGeometryRepresentationProxy_ in the 2D view might be the key. Do you have some insights to guide me on where and how I might be able to incorporate this? Any pointers or insights would be immensely helpful.

Best regards,

Thomas

**Comment by thogerig** [comment]
Thanks a lot. I have been able to make an working implementation for showing the slices. I have successfully used the mentioned proxy and added it to the TwoDView functionalities.

Best regards,

Thomas

**Reply by PaulHax** [reply]
Glad you figured it out!  If you can share, I'ed be curioius to hear more about the goals of your project.

**Reply by thogerig** [reply]
Thank you for your response. I am part of Shapemeans, a software company, based in Basel, Switzerland, that focuses on design automation utilizing statistical shape models. We are currently investigating methods for visualizing and managing shapes and images within web browsers.

**Reply by PaulHax** [reply]
Interesting work!  I hope VolView is helpful.

Another good place to ask questions: https://discourse.vtk.org/c/web/9
